### DISCUSSION TITLE
Export causing 419 (page expired)

### DISCUSSION THREAD

**Discussion by LemonPie22** [filamentphp/filament#11002] [Help] [answered]
I'm running Filament v3.2.7 locally and I'm using the new export feature. It works well but, after I click on the modal that allows me to select fields, I get this dialog:

<img width="454" alt="Screenshot 2024-01-23 at 11 24 32" src="https://github.com/filamentphp/filament/assets/107488019/5f1c5c53-397f-44e9-b2d5-faef42e55e6c">

And then this modal:

<img width="678" alt="Screenshot 2024-01-23 at 11 24 41" src="https://github.com/filamentphp/filament/assets/107488019/89256292-1445-4555-84c1-8cafa9956056">

This happens with debug on and off and I have disabled ad-blocker.
Any ideas?

**Comment by LemonPie22** [comment, marked as answer]
Seems to be fixed in v3.2.10

**Comment by olivsinz** [comment]
Try to add this in the VerifyCsrfToken middleware:
```
    protected $except = [
        'filament/exports/',
    ];
```

Inspired by : https://www.answeroverflow.com/m/1217738116371058739

**Comment by imsyuan** [comment]
In my environment (Filament v3.2.123), I have a total of 1,193 records.
Currently, I have added:

```
protected $except = [
    'filament/exports/',
];
```

However, I still encounter a 419 error.
I'm not sure what else I can adjust.
